app/views/shared/_email_form.html.erb

```diff
@@ -1,9 +1,10 @@
 <%= form_tag "#{path}", method: 'post' do %>
   <label class="usa-label margin-top-4 margin-bottom-2" for="email">Your email address</label>
   <input class="usa-input" id="email" name="email" type="email" required/>
-
   <label class="usa-label margin-top-4 margin-bottom-2" for="subject">Subject line</label>
   <input class="usa-input" id="subject" name="subject" type="text" required/>
+  <label class="usa-label form_h_field margin-top-4 margin-bottom-2" for="phone">Phone</label>
+  <input class="usa-input form_h_field" id="phone" name="phone" type="phone"/>
```

Comment:
I think we should change the param name to "phony" if that's what we're using for the column in the DB. It may be a little confusing down the line.

Reply:
I can change the db name to phone...  we want to keep the name 'phone' to look like a legit field.